### PR TITLE
Doc readme for dist

### PR DESCRIPTION
**Pull request by jklymak**
I guess I should figure out how to do this automatically, but at least have the right notes for now (suggestions welcome).

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -47,15 +47,18 @@ The built html pages can be found in ``doc/_build/html/``
...
+- be sure to edit `mpl_sphinx_theme/_version.py`
+
 .. code-block::
 
    $ git checkout <commit-hash>
    $ git tag -a x.y.z -m 'Version x.y.z'
    $ git push upstream main --tags
-   $ python -m build -s -w
+   $ python setup.py sdist
+   $ python setup.py bdist_wheel
```
Comment (QuLogic) [inline README.rst:61]: Did `build` not create both of them correctly? That is the new way going forward (to hide or even drop `setup.py` eventually.)

Comment (jklymak) [reply README.rst:61]: I guess I don't have build? What is it?

Comment (jklymak) [reply README.rst]: OK, `mamba install build` works fine ;-)

Comment (QuLogic) [reply README.rst]: It's a "PEP 517 build frontend", that is, it reads the build system defined in `pyproject.toml`, creates an isolated environment for it, and builds the sdist in there. So essentially it should work everywhere regardless of whether it's `setup.py`, or flit, or hatch, etc. Of course, it doesn't make too much difference for `setup.py`-based projects, but it should be more portable.

**Review by QuLogic — APPROVED** [review]

**Pull request merged**